config/clusters/2i2c-aws-us/enc-cosmicds.secret.values.yaml

```diff
@@ -1,22 +1,25 @@
 jupyterhub:
     hub:
+        services:
+            cosmicds-portal:
+                api_token: ENC[AES256_GCM,data:QY4XnFjLgb8KqvVcXignYeJg7Xi5S5IWZ9dAFYy326XFo/oeds7DSDdj2rKam6zRX00ZC1+Cvx+FHlhL0FsjYg==,iv:dgw2az+wRJMt4uwklBcuxaYK64ETnKYH8VX0vS6RzRs=,tag:GP2yWFm9u69iUCogRoCkoQ==,type:str]
         extraEnv:
-            USERNAME_DERIVATION_PEPPER: ENC[AES256_GCM,data:AXMgK5+Gzojb2j65OA87X0BEs4JxjZr1jkemLRNhMp5pxdvt40YyMEO2fyhx+nfNwrvMf9DV6z9Hl7l2XEsbTQ==,iv:B9EBaac4VFOkU+nzxcm7LUzqJRJ4N38o4BbsZqxW69Q=,tag:cERvKEh9TfxyoDyzzVrb1Q==,type:str]
+            USERNAME_DERIVATION_PEPPER: ENC[AES256_GCM,data:mH/CtGOgBMPqUfy74r1sOdxxS2UcUAs/JkSItBgNKguTkV6WmBkd1VvEiRbiFmh86/VZrgFMj9k3B+eUfBSvMw==,iv:ZBhhx3mMrydErwPTNRQ391yUcZo4UIaEooz3exBY3c8=,tag:4eTclvSgvqby9qthqjAEcA==,type:str]
         config:
             CILogonOAuthenticator:
-                client_id: ENC[AES256_GCM,data:bG3o+fgg9Un2YzPxgBisMSpZ7mu0NnF3u7fbHFf3TErMRSNZdbKYne9InZfntOSt9CFP,iv:K+L30QknUdByGTTgs/Xo7xdWAl3ceUjyRi09PFFq0Us=,tag:GRtn+QCLaBjToj4Wk42kEg==,type:str]
```

Comment:
Is the client id/secret change using CILogon as an identity provider an intended change?

Reply:
@consideRatio nope, and the information itself hasn't changed. I think sops re-encrypted here.